xml/ha_rear.xml

```diff
@@ -397,38 +399,18 @@
  </sect1>
  <sect1 xml:id="sec-ha-rear-config">
   <title>Setting up &rear; and your backup solution</title>
-
   <para>
-   To set up &rear;, you need to edit at least the &rear;
-   configuration file <filename>/etc/rear/local.conf</filename> and, if
-   needed, the Bash scripts that are part of the &rear; framework.
+   Before you can configure &rear;, you must install the latest &rear; package:
```

Comment:
"you must install the latest ReaR package"
is at least misleading because intentionally we provide
several ReaR versions in parallel so that users get the
freedom to choose what fits best their specific needs,
cf. above the part about
"To see which versions of ReaR are available..."

I think better wording is
"you must install a ReaR package - normally the highest available version unless otherwise required"

Reply:
I've gone with this: 

>  To install the latest version of ReaR, run the following command:
> `# zypper install rear`
> 
> If you need to install an earlier version of ReaR, you can specify the package version. For example:
> `# zypper install rear23a`